<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtJsonClass</class>
 <widget class="QWidget" name="QtJsonClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtJson</string>
  </property>
  <widget class="QPushButton" name="readJsonBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>读取Json文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="writeJsonBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>写入Json文件</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>361</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>信息窗口</string>
   </property>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>321</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtJson.qrc"/>
 </resources>
 <connections/>
</ui>
